packages/main/src/plugin/configuration-impl.ts

```diff
@@ -54,14 +55,70 @@ export class ConfigurationImpl implements containerDesktopAPI.Configuration {
   get<T>(section: string, defaultValue?: unknown): T | undefined {
     const localKey = this.getLocalKey(section);
 
-    // now look if we have this value
+    // Check if this config key is "locked" by a "managed-by" policy, and if so, return the "managed-by" value.
+    //
+    // Locked keys ignore user settings and force the managed default value instead, if there is a value there
+    // in the first place, if not, we fall back to the user value.
+    //
+    // We retrieve the information from from a system file:
+    // (ex macOS: /Library/Application Support/com.podman.desktop/locked.json, Linux: /usr/share/podman-desktop/locked.json)
+    //
+    // This only applies when we are getting values from the default scope, as getting them SPECIFICALLY
+    // from managed defaults or locked scopes would be counter-intuitive... (or Onboarding scope, but that doesn't make sense either)
+    if (this.scope === CONFIGURATION_DEFAULT_SCOPE) {
+      // Pass in the key we are wanting to get and check locked config for the key
+      const managedValue = this.getLockedConfigValue<T>(localKey);
+
+      // Before we go ahead and return the value, we want to log to console
+      // this information so that if users are confused why they can't change a setting, they know why,
+      // however, this information should only stay in console.debug considering that it could be noisy (returning configuration values multiple times)
+      //
+      // A GUI indication will be provided in the settings page for locked values:
+      // https://github.com/podman-desktop/podman-desktop/issues/13900
+      if (managedValue !== undefined) {
+        console.debug(
```

Comment:
it will produce way too many log, IMHO should be removed

Also the [Managed-By] is very confusing as it's dispatched in too many locations.

Reply:
ACK, I agree